tests/input_data/test_emr.py

```diff
@@ -0,0 +1,194 @@
+        "standard_error": [0.0005, 0.0004, 0.0003, 0.0002, 0.0001] * 2,
+    }
+)
+
+SPAN_PREVALENCE = pd.DataFrame(
+    {
+        "age_lower": [0, 3, 20, 50, 70, 80],
+        "age_upper": [0.001, 4, 25, 55, 90, 80],
+        "time_lower": [1990, 1995, 2000, 2005, 2010, 2015],
+        "time_upper": [1995, 2000, 2005, 2010, 2015, 2020],
+        "sex_id": [3] * 6,
+        "node_id": [6] * 6,
+        "density": [DensityEnum.gaussian] * 6,
+        "weight": ["constant"] * 6,
+        "mean": [0.01, 0.02, 0.03, 0.04, 0.05, 0.06],
+        "standard_error": [0.005, 0.004, 0.003, 0.002, 0.001, 0.001],
+        "measure": ["prevalence"] * 6,
+    }
+)
+
+
+def test_interpolators__points_only():
+    mean, stderr = _make_interpolators(POINT_CSMR)
+
+    assert mean["both"](0, 1990) == 0.006
+    assert mean["both"](1, 1990) == 0.007
+    assert mean["both"](10, 1990) == 0.008
+    assert mean["both"](15, 1990) == 0.009
+    assert mean["both"](20, 1990) == 0.01
+
+    assert stderr["both"](0, 1990) == 0.0005
+    assert stderr["both"](1, 1990) == 0.0004
+    assert stderr["both"](10, 1990) == 0.0003
+    assert stderr["both"](15, 1990) == 0.0002
+    assert stderr["both"](20, 1990) == 0.0001
+
+
+def test_emr_from_sex_and_node_specific_csmr_and_prevalence__perfect_alignment__points_only():
+    emr = _emr_from_sex_and_node_specific_csmr_and_prevalence(POINT_CSMR, POINT_PREVALENCE)
+
+    emr = emr.set_index(["age_lower", "age_upper", "time_lower", "time_upper", "sex_id", "node_id"])
+    csmr = POINT_CSMR.set_index(["age_lower", "age_upper", "time_lower", "time_upper", "sex_id", "node_id"])
+    prevalence = POINT_PREVALENCE.set_index(["age_lower", "age_upper", "time_lower", "time_upper", "sex_id", "node_id"])
+
+    assert len(emr) == len(prevalence)
+
+    assert np.allclose(emr["mean"], csmr["mean"] / prevalence["mean"])
+
+
+def test_emr_from_sex_and_node_specific_csmr_and_prevalence__perfect_alignment__spans():
+    a = -0.01
+    b = -0.02
+    c = 1.0
+    csmr_mean = lambda age, time: -(-a * 0 - b * 1980 - c * 0.001 + a * age + b * time) / c
+
+    csmr_rows = []
+    for age in range(0, 120):
+        for time in range(1980, 2040):
+            csmr_rows.append(
+                {
+                    "age": age,
+                    "time": time,
+                    "sex_id": 3,
+                    "node_id": 6,
+                    "mean": csmr_mean(age, time),
+                    "standard_error": 0.01,
+                }
+            )
+
+    csmr = pd.DataFrame(csmr_rows)
+
+    emr = _emr_from_sex_and_node_specific_csmr_and_prevalence(csmr, SPAN_PREVALENCE)
+
+    assert len(emr) == len(SPAN_PREVALENCE)
+
+    for (_, pr), (_, er) in zip(SPAN_PREVALENCE.iterrows(), emr.iterrows()):
+        pmean = pr["mean"]
+        cmean = csmr_mean((pr["age_lower"] + pr["age_upper"]) / 2, (pr["time_lower"] + pr["time_upper"]) / 2)
+        assert er["mean"] - cmean / pmean < er["standard_error"]
```

Comment:
Does this pass now?

Reply:
Don't ask me, ask Travis. Which is to say, yes. That is the right way to loosen the test, right?